src/components/PageElements/ViewHistoryButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Label } from 'semantic-ui-react'
+import strings from '../../utils/constants'
+
+const ViewHistoryButton: React.FC = () => {
+  return (
+    <div className="history-container">
+      <Label
+        className="link-label clickable"
+        icon="history"
```

Comment:
Just my preference, but I think the icon could be a bit bigger.

Reply:
No problem I'll use the same size as other icons

Reply:
Done